lib/allowlist.js

```diff
@@ -22,11 +22,17 @@ class Allowlist {
     }
     input.forEach((arg) => {
       if (typeof arg === "number") {
-        this.advisories.push(arg);
-      } else if (arg.includes(">") || arg.includes("|")) {
+        throw new Error(
+          "Unsupported number parameter - Ref https://github.com/IBM/audit-ci/pull/217"
+        );
+      }
```

Comment:
We could of course also silently ignore string parameters here, but I think it would be better to tell people that they are doing something wrong here.